robot_framework/eflyt.py

```diff
@@ -257,32 +257,55 @@ def approve_case(browser: webdriver.Chrome):
 
 def get_age(cpr: str) -> int:
     """Get the age of a person based on their cpr number
-    assuming they are between 0-99 years old.
+    using the 7th digit to infer the century.
 
     Args:
-        cpr: The cpr number in the format 'ddmmyy-xxxx' or 'ddmmyyxxxx'.
+        cpr: The cpr number in the format 'ddmmyyxxxx'.
 
     Returns:
         The age based on the cpr number.
     """
+    # A dictionary mapping from a year (0-99) and control digit (0-9) to a century.
+    # https://cpr.dk/media/12066/personnummeret-i-cpr.pdf
+    # The keys of the dict corespond to the control digit.
+    # The first value of the tuples is the cutoff year. If the input year is equal to or below the first value
+    # the first century is used. If the year is above the first value, the second century is used.
+    # E.g. If the control is 4 and the year is smaller or equal to 36 -> 2000-2036
```

Comment:
It may be worth adding a another example for 4:

```suggestion
    # E.g. If the control is 4 and the year is less than or equal to 36 -> 2000-2036
    # E.g. If the control is 4 and the year is greater than 36 -> 1937-1999
```